bukuserver/templates/bukuserver/lib.html

```diff
@@ -11,7 +11,7 @@
 {% endmacro %}
 
 {% macro close_if_popup() %}
-  <script>opener && (opener !== window) && close()</script>
+  <script>({{ g.popup|default|tojson }} || (opener && (opener !== window))) && close()</script>
```

Comment:
Reading the detected `popup=` parameter (with fallback to `''`) in pages that should be closed if navigated to from within the popup ("Save"/"Delete" behaviour)